### PR TITLE
CfD Vimeo video added

### PR DESCRIPTION
**Pull request by egdavis**
New component: VimeoVideo
Changes to: WhoWeAre and style.css

The VimeoVideo now appears on the mainpage for the website.

### CODE REVIEW

File: static/css/style.css [hunk 1]
```diff
@@ -563,6 +567,10 @@ blockquote
     left:0;
   }
 */
+  .vimeo-video
+  {
+    display: none;
```
Comment (wdoug) [inline static/css/style.css]: :question: Why are we hiding it?

File: src/components/segments/VimeoVideo.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class Video extends React.Component {
+  render() {
+    return (
+    	<div className="vimeo-video">
+      		<iframe src="https://player.vimeo.com/video/126984194" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```
Comment (wdoug) [inline src/components/segments/VimeoVideo.js]: Seems like we'll want a responsive player. Here is a way to do that that I recently came across: http://jsfiddle.net/masau/7wrhm/

File: static/css/style.css (hunk 1 above)
Comment (egdavis) [inline static/css/style.css]: It just looked funny to me when I made the window smaller. Maybe you can show me what a better solution would look like.

File: src/components/segments/VimeoVideo.js (hunk 2 above)
Comment (egdavis) [inline src/components/segments/VimeoVideo.js]: I am not familiar with anything like that. But I will take a look at your suggestion.

**Comment by wdoug** [comment]
What about adding these options to the styling of the video: https://player.vimeo.com/video/126984194?color=ffffff&byline=0&portrait=0&badge=0

File: static/css/style.css [hunk 3]
```diff
@@ -393,6 +393,22 @@ blockquote
...
+  height: 0;
+}
+.vimeo-video iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  border: none;
+  margin: 1rem;
```
Comment (wdoug) [inline static/css/style.css]: Let's move this margin to a container element

**Pull request merged**